packages/workspace/src/core/affected-project-graph/locators/implicit-json-changes.ts

```diff
@@ -0,0 +1,46 @@
+import { NxJson } from '../../shared-interfaces';
+import { FileChange } from '../../file-utils';
+import { JsonValueDiff } from '../../../utils/json-diff';
+
+export function getImplicitlyTouchedProjectsByJsonChanges(
```

Comment:
Is this the one that identifies what is affected when `[package.json].scripts.deploy` is touched?

Reply:
Bingo!

Reply:
But this works for any JSON pathing at the first level? I feel like we'll need more levels for later but this is good for now.

Reply:
It will work any level deep actually! So this would work:

```
{
  implicitDependencies: {
    'foo.json': {
      a: {
        b: {
          c: {
            d: '*'
          }
        }
      }
    }
}
```

Reply:
Awesome!